helm-charts/orchestrator/templates/tekton-pipeline.yaml

```diff
@@ -130,7 +142,7 @@ spec:
         - name: CONTEXT
           value: flat/$(params.workflowId)
         - name: BUILD_EXTRA_ARGS
-          value: '--authfile=/workspace/dockerconfig/.dockerconfigjson --ulimit nofile=4096:4096 --build-arg WF_RESOURCES="." '
+          value: '--authfile=/workspace/dockerconfig/.dockerconfigjson --ulimit nofile=4096:4096 --build-arg WF_RESOURCES=. '
```

Comment:
just to make sure: this change works for both versions?

Reply:
should be as the authfile as no quotes, but I'll test in 1.16 just to be sure

Reply:
Just checked, it works without the quotes in 1.16